nginx-plus/etc/nginx/conf.d/status_api.conf

```diff
@@ -75,7 +75,7 @@ server {
     # Conventional location for accessing the NGINX Plus API 
     location /api/ {
         # Enable in read-write mode
-        # api write=on;
+        api write=on;
```

Comment:
I dont think we need write access to API ?

Reply:
I added it to enable the dashboard but I see your point that it can be read only